alignment.Dockerfile

```diff
@@ -0,0 +1,62 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:focal
+ENV DEBIAN_FRONTEND=noninteractive
+RUN <<EOF
+# build SDL2
+# define installation directory
+mkdir SDL2
+export SDL2_INSTALLATION=/SDL2
+# clone SDL
+apt-get update -y
+apt install -y git build-essential
+git clone https://github.com/libsdl-org/SDL.git -b SDL2 && cd SDL && git checkout tags/release-2.26.2
+# build
+mkdir build && cd build && ../configure --prefix=$SDL2_INSTALLATION --enable-video-offscreen && make && make install
+EOF
+
+RUN <<EOF
+# build Irrlicht
+# install missing dependency
+apt-get update -y
+apt install -y zlib1g-dev libjpeg-dev libpng-dev libxi-dev cmake
+# clone irrlicht
+git clone https://github.com/EleutherAI/irrlicht && cd irrlicht && git checkout disable-x11
+# define lib paths
+export SDL2_INSTALLATION=/SDL2
+# build
+cmake . -DBUILD_SHARED_LIBS=OFF -DBUILD_HEADLESS=1 -DSDL2_DIR=$SDL2_INSTALLATION/lib/cmake/SDL2
+make -j8
+EOF
+
+RUN <<EOF
+# build minetest
+# install dependencies
+apt-get update -y
+apt install -y libzmqpp-dev libgmp3-dev protobuf-compiler libprotobuf-dev libzstd-dev libfreetype-dev libsqlite3-dev
+# clone minetest fork
+git clone https://github.com/EleutherAI/minetest
+git clone --depth 1 https://github.com/minetest/minetest_game.git minetest/games/minetest_game
+cd minetest
+git checkout cmake-SDL2-target-fix
```

Comment:
this branch was just for testing SDL.
I guess for now this should be `git checkout develop`